client/dan.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+  <meta name="generator" content="Hugo 0.72.0" />
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:title" content="Daniel Posse" />
+  <meta name="twitter:description" content="" />
+
+  <title>Daniel Posse</title>
+  <link rel="alternate" type="application/rss+xml" href="http://localhost:1313/index.xml" title="Daniel Posse" />
+  <link rel="stylesheet" href="/css/dan.css">
+</head>
+
+<body id="page">
+
+  <div id="spotlight" class="animated fadeIn">
+    <div id="home-center">
+      <h1 id="home-title">Daniel Posse</h1>
+      <p id="home-subtitle">Full Stack Software Engineer | Doughnut Enthusiast | Dog Lover</p>
+      <div id="home-social">
+        <a href="https://github.com/dposse" target="_blank" rel="noopener me" title="Github"><svg
+            xmlns="http://www.w3.org/2000/svg" class="feather" width="24" height="24" viewBox="0 0 24 24" fill="none"
+            stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+            <path
+              d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22">
+            </path>
+          </svg></a><a href="https://www.linkedin.com/in/daniel-posse/" target="_blank" rel="noopener me"
+          title="Linkedin"><svg xmlns="http://www.w3.org/2000/svg" class="feather" width="24" height="24"
+            viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+            stroke-linejoin="round">
+            <path d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z"></path>
+            <rect x="2" y="9" width="4" height="12"></rect>
+            <circle cx="4" cy="4" r="2"></circle>
+          </svg></a><a href="mailto:dan.e.posse@gmail.com" target="_blank" rel="noopener me" title="Email"><svg
+            xmlns="http://www.w3.org/2000/svg" class="feather" width="24" height="24" viewBox="0 0 24 24" fill="none"
+            stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+            <path d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"></path>
+            <polyline points="22,6 12,13 2,6"></polyline>
+          </svg></a>
+      </div>
+    </div>
+
+    <div>
+      <h3>The above is what my <a href="https://dposse.dev" target="_"><u>personal site</u></a> <em>should</em> look
+        like...
+        but there's
+        some
+        silly CORS
```

Comment:
See this for fastest quick and dirty CORS workaround: https://github.com/izuzak/urlreq

See an example of how I've used it in the past: https://www.karstenrabe.dev/#github-scraper-code